express-static/public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Static Middleware</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <h1>Static Middleware!</h1>
+    <script src="/main.js"></script>
+    <img
+      src="./images/3445-3840x2160-desktop-4k-lamborghini-wallpaper-image.jpg"
```

Comment:
These src paths should _not_ have a leading '.'. They should just be '/images/file-name'.